weave/flow/eval.py

```diff
@@ -370,7 +393,7 @@ async def predict_and_score(
                     for param in score_signature.parameters.values()
                     if param.default == inspect.Parameter.empty
                 ]
-                required_arg_names.remove("output")
+                required_arg_names.remove(self._output_key)
```

Comment:
this should dynamic based on the scorer

Reply:
I think this should be `required_arg_names.remove(score_output_name)` instead or we could just remove this line.